tracer/src/Datadog.Trace.Tools.dd_dotnet/JsonConfigurationSource.cs

```diff
@@ -0,0 +1,47 @@
+// <copyright file="JsonConfigurationSource.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Text.Json;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.Tools.dd_dotnet;
+
+internal class JsonConfigurationSource : IConfigurationSource
+{
+    private readonly JsonDocument _document;
+
+    public JsonConfigurationSource(string json)
+    {
+        _document = JsonDocument.Parse(json);
```

Comment:
System.Text.Json has significantly different defaults to Newtonsoft.Json, e.g. around how lax it is with trailing commas, comments, that sort of thing. We should probably use lax defaults here, to best match the behaviour in the app?